sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java

```diff
@@ -26,6 +26,7 @@
   public static final By UUID_INPUT = By.id("uuid");
   public static final By TITLE_INPUT = By.cssSelector("#eventTitle");
   public static final By SAVE_BUTTON = By.id("commit");
+  public static final By SAVE_BUTTON_POPUP = By.cssSelector(".popupContent #commit");
```

Comment:
this button is from Pick or create event pop-up, can we rename it to be more clear? :D

Reply:
for both but its also works for the new task, new action, link subordinate event or do want to create a specific one with xpath for each ?

Reply:
no no, that is the name of the pop-up : **Pick or create event pop-up** 
since the locator is on a specific page, and the popup has a specific name, to avoid duplicates in case of multiple page object pop-ups, we can be a bit more specific with this save button :D

Reply:
all the locators mentioned above are in the same edit event page, renamed to SAVE_BUTTON_FOR_POPUP_WINDOWS i don't think making a new page just for 1-2 locators for a specific popup window is better, but if its needed then I will :)